sentry-core/src/main/java/io/sentry/core/SentryThreadFactory.java

```diff
@@ -53,7 +53,9 @@ private SentryThread getSentryThread(
     sentryThread.setDaemon(thread.isDaemon());
     sentryThread.setState(thread.getState().name());
     if (crashedThreadId != null) {
+      // TODO: do we still need crashed?
       sentryThread.setCrashed(crashedThreadId == thread.getId());
+      sentryThread.setErrored(sentryThread.isCrashed());
```

Comment:
is it here the right place to mark? or the `else` case? it's a bit confusing as we have `crashed, daemon, current and errored`